src/qt/bitcoingui.cpp

```diff
@@ -405,6 +415,8 @@ void BitcoinGUI::createMenuBar()
     file->addAction(signMessageAction);
     file->addAction(verifyMessageAction);
     file->addSeparator();
+    file->addAction(snapshotAction);
+    file->addSeparator();
```

Comment:
This needs to be conditioned on testnet to address @cyrossignol's comments I think.

Reply:
looking at this now

Reply:
yes i moved the check there and then checked testnet argument since fTestNet will not be set yet since gui fast loads